hack/jenkins/test-dockerized.sh

```diff
@@ -33,7 +33,7 @@ export GO111MODULE=off
 
 # Install tools we need
 pushd "./hack/tools" >/dev/null
-  GO111MODULE=on go install gotest.tools/gotestsum
+  GOTOOLCHAIN="${KUBE_HACK_TOOLS_GOTOOLCHAIN:-"${GOTOOLCHAIN}"}" GO111MODULE=on go install gotest.tools/gotestsum
```

Comment:
this one fails because we haven't actually called `kube::golang::setup_env` yet, so GOTOOLCHAIN is unset

Reply:
bah... the one we needed most 🤦

Reply:
we should probably do the subshell-if form, so we just don't set GOTOOLCHAIN if neither are set.

Reply:
could just fall all the way back to empty with ${GOTOOLCHAIN:-} as well